docs/3.x/tech/installation-guide/docker.adoc

```diff
@@ -35,13 +35,13 @@ In the above example configuration, you will find a database, search, and CleanS
 
 === Database Service
 
-At a minimum, you will need to either set the POSTGRES_PASSWORD environment variable in the DB service section or, more ideally, set the value in the host environment and leave it out of the docker-compose.yml file. Ensure the other properties fit your requirements. Refer to the link:system-requirements[System Requirements] for database version support.
+At a minimum, you will need to either set the `POSTGRES_PASSWORD` environment variable in the DB service section or, more ideally, set the value in the host environment and leave it out of the docker-compose.yml file. Ensure the other properties fit your requirements. Refer to the link:system-requirements[System Requirements] for database version support.
```

Comment:
Some files you are using back ticks, and others you are not. Should we use them on `docker-compose.yml`?